Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -1618,27 +1620,28 @@ def show_paper_datatables(self, window, paper):
                 for k in keys:
                     (header, data) = paper.datatables[k]
                     t_frame = Tkinter.Frame(frame)
-                    l = Tkinter.Label(t_frame, text=header, font=HEADING_FONT)
+                    lbl = Tkinter.Label(t_frame, text=header,
+                                        font=HEADING_FONT)
```

Comment:
@konvpalto,
and talking about "whitespace-only change does not belong here" -- then does the line breaking one belong?

Reply:
It is about "Whitespace-only: belongs to elsewhere", yes?  My wording is ambigious (and, thus, bad): what I meant is "belongs to another commit", not the "another pull".

Whitespace-only fixes with not functional changes are OK to go along with anything, especially if they correspond to the same lines of code (or touch some of them) that were fixing the functionality/code-style related ones, in my view.  They just should be marked as such, nothing more.